src/components/symptom-shark/presentational/OverallExperienceChart/OverallExperienceChart.tsx

```diff
@@ -77,7 +78,7 @@ export default function (props: OverallExperienceChartProps) {
 
     let overallFeelingAverage = calculateOverallFeelingAverage();
 
-    return <div ref={props.innerRef} className="mdhui-ss-oe-chart">
+    return <div ref={props.innerRef} className={"mdhui-ss-oe-chart" + (props.variant == "monthReport" ? " mdhui-ss-oe-chart-month-report" : "")}>
```

Comment:
extra space " mdhui-ss-oe-chart-month-report" vs "mdhui-ss-oe-chart-month-report"

Reply:
Pretty sure the space is purposeful so that the resulting generated string has two distinct CSS class names being applied to it, both `mdhui-ss-oe-chart` *and* `mdhui-ss-oe-chart-month-report`.  Without the space it would look like one extra long css class name which is incorrect.